src/Bicep.Core.IntegrationTests/LoopInvariantTests.cs

```diff
@@ -65,7 +66,13 @@ public void VariantResourcePropertiesShouldProduceNoWarning()
 ";
 
             var result = CompilationHelper.Compile(text);
-            result.Should().NotHaveAnyDiagnostics();
+            result.Should().HaveDiagnostics(new[]
```

Comment:
>HaveDiagnostics [](http://example.com/codeflow?start=28&length=15)

Weren't we going to disable the linter in non-related tests and just leave it for the baseline tests?

Reply:
I see it's fixed now.

Reply:
Removed because rules were removed for 0.4.  Need to discuss approach when we add back those rules.

Reply:
I guess we'll discuss when we get to that point.

Reply:
Weren't we going to disable the linter in non-related tests and just leave it for the baseline tests?